src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2317,12 +2317,21 @@ public static void TimeZoneInfo_DisplayNameStartsWithOffset()
                     {
                         string offset = Regex.Match(tzi.DisplayName, @"(-|)[0-9]{2}:[0-9]{2}").Value;
                         TimeSpan ts = TimeSpan.Parse(offset);
-                        if (tzi.BaseUtcOffset != ts && tzi.Id.IndexOf("Morocco", StringComparison.Ordinal) >= 0)
+                        if (tzi.BaseUtcOffset != ts && (tzi.Id.IndexOf("Morocco", StringComparison.Ordinal) >= 0 || tzi.Id.IndexOf("Volgograd", StringComparison.Ordinal) >= 0))
```

Comment:
Nit, I guess this could check `PlatformDetection.IsWindows` as well perhaps.

Reply:
I can add it this condition. I don't recall we never got any failure on Linux.